movie-subdomain/src/main/kotlin/com/example/ajaxonboardingproject/application/repository/MovieRepository.kt

```diff
@@ -1,6 +1,6 @@
-package com.example.ajaxonboardingproject.repository
+package com.example.ajaxonboardingproject.application.repository
```

Comment:
if you want to access from application something that should be in infrastructure - you can provide OutPort interface which will be implemented in infrastructure.

Actually you did this, but let's name it MovieRepositoryOutPort to understand it's nature. and in MovieRepositoryImpl you can omit impl suffix

Reply:
Done! ✅